src/librustdoc/json/conversions.rs

```diff
@@ -33,7 +33,7 @@ impl JsonRenderer<'_> {
             id: from_def_id(def_id),
             crate_id: def_id.krate.as_u32(),
             name: name.map(|sym| sym.to_string()),
-            source: self.convert_span(span),
+            span: self.convert_span(span),
```

Comment:
This is a breaking change, I don't think it belongs here.

Reply:
I thought the idea was to change the JSON API now, before it's stable?

Reply:
Hmm, I guess it's fine if you bump format-version. This change doesn't improve the api a ton which is why I'm a little skeptical, but I guess the alternative is for it to be different from the compiler in perpetuity. 

cc @craftspider @heroickatora just in case

Reply:
Thanks for the notification, :+1: from me with format-version bumped.

Reply:
Done.

Reply:
:+1: from me as well, the `Import.span` change was on my radar as well, I approve the name swap to match the actual intentions of these fields.